recipe/build_base.bat

```diff
@@ -4,6 +4,11 @@ set
 
 set "MESON_ARGS=%MESON_ARGS% --buildtype=release --prefix=%LIBRARY_PREFIX% --pkg-config-path=%LIBRARY_LIB%\pkgconfig -Dlibdir=lib -Dsystem-freetype=true -Dsystem-qhull=true"
 
+if "%CI%" == "azure" (
+    :: Hack to try removing problematic Python from Azure CI image
+    rmdir /s /q C:\hostedtoolcache\windows\Python
+)
```

Comment:
This probably shouldn't stay here. Just want to see what happens if we remove it altogether. If this works, probably we move this to conda-smithy

Reply:
Seems like it worked at least. I can't see any mention of this path in the Meson logs though.

Reply:
Indeed. Odd that it was picked up in the first place, but certainly a viable solution